examples/advanced/xgboost/histogram-based/requirements.txt

```diff
@@ -2,3 +2,5 @@ nvflare>=2.3.0
 pandas
 xgboost>=2.0.0
 scikit-learn
+torch
+tensorboard
```

Comment:
What are we visualizing here?
I did not find where we write out metrics, can you point that out?
Do we need torch?

Reply:
torch is used in utils/baseline_centralized.py